src/main/java/com/conveyal/r5/analyst/TravelTimeComputer.java

```diff
@@ -75,11 +76,16 @@ public OneOriginResult computeTravelTimes() {
         // TODO Create and encapsulate this object within the propagator.
         TravelTimeReducer travelTimeReducer = new TravelTimeReducer(request);
 
-        // Determine car pick-up delay time for the access leg, which is generally specified in a scenario modification.
-        // Only find this time when cars are in use, as it requires potentially slow geometry operations.
-        // Negative values mean no car service is available at the origin location.
-        final int carPickupDelaySeconds = (request.accessModes.contains(LegMode.CAR)) ?
-            network.streetLayer.getWaitTime(request.fromLat, request.fromLon) : 0;
+        // Look up pick-up delay time for the access leg, which is generally specified in a scenario modification.
+        // Only find this time if CAR or BICYCLE_RENT is an access mode, as this lookup requires potentially slow
+        // geometry operations. Negative values mean no pickup service is available at the origin location.
+        final int pickupDelaySeconds;
+
+        if (request.accessModes.contains(LegMode.CAR) || request.accessModes.contains(LegMode.BICYCLE_RENT)){
```

Comment:
I'm in favor of specifying an array of `StreetModes` in the modification.

Reply:
We just discussed this, and can't think of any reason the same wait times would apply to both car and bicycle. So we'll require exactly one mode to be specified in a pickup-delay modification, and it will only apply to that mode.